cmd/oci/main.go

```diff
@@ -50,5 +60,48 @@ func main() {
 	log.SetOutput(f)
 
 	log.Printf("OCI FlexVolume Driver version: %s (%s)", version, build)
-	flexvolume.ExecDriver(&driver.OCIFlexvolumeDriver{}, os.Args)
+
+	registerDrivers()
+
+	driver, err := getDriverFromArgs()
+	if err != nil {
+		log.Fatalf(err.Error())
+	}
+
+	flexvolume.ExitWithResult(flexvolume.ExecDriver(driver, os.Args))
+}
+
+func getDriverFromArgs() (flexvolume.Driver, error) {
```

Comment:
```go
func getDriverName(args []string) string
```
Calling code would then just call `getDriver(getDriverName(os.Args))`.

Would better facilitate unit testing (a quick table driven test would be desirable).